app/next-client-app/types/index.ts

```diff
@@ -13,3 +13,27 @@ interface ScanReport {
   previous: string | null;
   results: ScanReportResult[];
 }
+
+interface DataSet {
+  count: number;
+  next: string | null;
+  previous: string | null;
+  results: DataSetResult[];
+}
+
+interface DataSetResult {
+  id: number;
+  name: string;
+  hidden: boolean;
+  visibility: string;
+  created_at: string;
+  data_partner: DataPartnerData;
+  admins: string[];
+}
+
+interface DataPartnerData {
```

Comment:
And this can be just called a `DataPartner`


a nice read on naming things: https://journal.stuffwithstuff.com/2009/06/05/naming-things-in-code/
s/o @beforan